ecs_logging/_utils.py

```diff
@@ -151,17 +151,46 @@ def json_dumps(value):
         # case the given values aren't strings (even though
         # they should be according to the spec)
         ordered_json = ",".join(
-            '"%s":%s' % (k, json.dumps(v, sort_keys=True, separators=(",", ":")))
+            '"%s":%s'
+            % (
+                k,
+                json.dumps(
```

Comment:
Starting to be a lot of duplication in this function, maybe we need to wrap `json.dumps(..., <all-of-our-options>)` somewhere? Can be an inline function or `functools.partial`?

Reply:
Done.